resources/views/index.blade.php

```diff
@@ -9,92 +9,25 @@
-
-            <table class="data-table">
-                <thead>
-                    <tr>
-                        @foreach ($columns as $column)
-                            <th>{{ $column['title'] }}</th>
-                        @endforeach
-                        <th class="actions-column"></th>
-                    </tr>
-                </thead>
-
-                <tbody>
-                    @foreach($records as $record)
-                        <tr>
-                            @foreach ($columns as $column)
-                                <td>
-                                    @if($column['has_link'])
-                                        <div class="flex items-center">
-                                            <a href="{{ cp_route('runway.edit', [
-                                                'resourceHandle' => $resource->handle(),
-                                                'record' => $record->{$resource->routeKey()}
-                                            ]) }}">{{ $column['fieldtype']->preProcessIndex($record->{$column['handle']}) }}</a>
-                                        </div>
-                                    @else
-                                        {!! $column['fieldtype']->preProcessIndex($record->{$column['handle']}) !!}
-                                    @endif
-                                </td>
-                            @endforeach
-
-                            <td class="flex justify-end">
-                                <dropdown-list>
-                                    <dropdown-item text="Edit" redirect="{{ cp_route('runway.edit', ['resourceHandle' => $resource->handle(), 'record' => $record->{$resource->routeKey()}]) }}"></dropdown-item>
-                                    <form action="{{ cp_route('runway.destroy', ['resourceHandle' => $resource->handle(), 'record' => $record->{$resource->routeKey()}]) }}" method="POST">
-                                        @csrf
-                                        @method('DELETE')
-
-                                        <dropdown-item class="warning" text="Delete" redirect="#"></dropdown-item>
-                                    </form>
-                                </dropdown-list>
-                            </td>
-                        </tr>
-                    @endforeach
-                </tbody>
-            </table>
-        </div>
-
-        <div class="my-2">
-            {{ $records->links('runway::pagination') }}
-        </div>
-    @else
+    @if ($recordCount > 0)
+    <runway-listing-view
+        :filters="{{ $filters->toJson() }}"
+        :listing-config="{{
+            collect([
+                'preferencesPrefix' => 'oopla.bonuses',
+                'requestUrl' => 'runway/'.$resource->handle().'/api',
+                'editUrl' => 'runway/'.$resource->handle(),
+                'deleteUrl' => 'runway/'.$resource->handle(),
+                'listingUrl' => 'runway/'.$resource->handle(),
```

Comment:
Would you mind hooking these URLs in to use the `cp_route` method (if possible), I don't love having URLs hard coded in places. Just personal preference.

Reply:
Done - let me know if I've understood correctly.